images/base/Dockerfile

```diff
@@ -12,6 +12,8 @@ FROM centos:centos7
 # base images land
 RUN yum swap -y -- remove systemd-container\* -- install systemd systemd-libs
 
-RUN yum install -y git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
+RUN yum install --setopt=tsflags=nodocs -y git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
     socat ethtool device-mapper iptables && \
-    yum clean all
+    yum clean all && \
+    rpm -V git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
```

Comment:
@bparees @sdodson we can set that env for this RUN command, so we don't need to have extra layer, what you think?

```
RUN install_pkgs="foo bar" && yum install -y $install_pkgs && rpm -V $install_pkgs
```